tanagra-aou-utils/run_tanagra_server.sh

```diff
@@ -30,11 +30,11 @@ export TANAGRA_FEATURE_ARTIFACT_STORAGE_ENABLED=true
 export TANAGRA_AUTH_IAP_GKE_JWT=false
 
 if [[ ${disableAuthChecks} ]]; then
-  echo "Disabling auth checks."
+  echo "Disabling AuthZ"
```

Comment:
Please change to Authentication (AuthN). See below

Reply:
Done